### PR TITLE
Feature/tc synth new model

### PR DESCRIPTION
**Pull request by mmyrte**
Changes proposed in this PR:
- Adds logic for intensification/peaking/decay over ocean
- Incorporates existing logic for landfall decay

This PR is for now intended to coordinate between @bguillod and myself.

### Pull Request Checklist

- [ ] Correct target branch selected (if unsure, select `develop`)
- [ ] Source branch up-to-date with target branch
- [ ] Docs updated
- [ ] Tests updated
- [ ] Tests passing
- [ ] No new linter issues

### CODE REVIEW

File: climada/conf/climada.conf [hunk 1]
```diff
@@ -42,10 +42,6 @@
         }
     },
     "hazard": {
-        "relative_cropyield": {
```
Comment (mmyrte) [inline climada/conf/climada.conf:45]: shit, don't know how this came about. should be removed from this branch, if possible.

File: climada/engine/unsequa/unc_output.py [hunk 2]
```diff
@@ -962,7 +962,7 @@ def plot_sensitivity_map(self, salib_si='S1', **kwargs):
         eai_max_si_df = self.get_largest_si(salib_si, metric_list=['eai_exp'])
 
         plot_val = eai_max_si_df['param']
-        coord = np.array([self.coord_df.latitude, self.coord_df.longitude]).transpose()
```
Comment (mmyrte) [inline climada/engine/unsequa/unc_output.py:965]: again, should remove this from the branch

File: climada/hazard/tc_tracks_synth.py [hunk 3]
```diff
@@ -342,6 +439,21 @@ def _one_rnd_walk(track, nb_synth_tracks, max_shift_ini, max_dspeed_rel, max_ddi
         i_track.attrs['sid'] = f"{i_track.attrs['sid']}_gen{i_ens + 1}"
         i_track.attrs['id_no'] = i_track.attrs['id_no'] + (i_ens + 1) / 100
         i_track = i_track.isel(time=slice(None, last_idx))
+        if adjust_intensity:
+            # compute minimum pressure occurring on or after each timestep
```
Comment (mmyrte) [inline climada/hazard/tc_tracks_synth.py:443]: @bguillod This is where I'd implement the fitting of MSW and radii to p_central.

File: climada/hazard/tc_tracks_synth.py [hunk 4]
```diff
@@ -71,17 +80,38 @@
...
+    nb_synth_tracks: int = 9,
+    max_shift_ini: float = 0.75,
+    max_dspeed_rel: float = 0.3,
+    max_ddirection: float = np.pi / 360,
+    autocorr_dspeed: float = 0.85,
+    autocorr_ddirection: float = 0.5,
+    seed: int = CONFIG.hazard.trop_cyclone.random_seed.int(),
+    adjust_intensity: bool = True,
+    central_pressure_pert: float = 7.5,
+    legacy_decay: bool = False,
```
Comment (bguillod) [inline climada/hazard/tc_tracks_synth.py:111]: Why did you rename `decay` to `legacy_decay`? The way I has implemented it in my first commits is that `decay` was a legacy parameter, however if a user set it they get a warning to use `adjust_intensity` instead and that parameter value is overwritten by `decay`.

**Comment by bguillod** [comment]
Closing this until the model is ready.
